crates/derive/src/stages/attributes_queue.rs

```diff
@@ -143,7 +143,7 @@ where
         attributes.no_tx_pool = true;
         attributes.transactions.extend(batch.transactions);
 
-        info!(
+        debug!(
```

Comment:
This one too - why `debug` now?

Reply:
It's very verbose when running trusted sync and I don't really consider these logs as info - it's really a trace or a debug. It provides introspection if you want to increase the verbosity level, but it doesn't provide useful information to downstream consumers of the pipeline I think. It just clutters logs.